data/merchant intro.txt

```diff
@@ -0,0 +1,138 @@
+			choice
+				`	"I'll do it."`
+					accept
+				`	"I don't have the time."`
+					decline
+	on complete
+		payment 40000
+		conversation
+			`The family thanks you profusely, grateful for meeting such a generous captain. You watch the family walk off and exchange greetings with another man in a blue suit, the blue-suited man walks up to you and says, "Meet me in the spaceport, don't dally."`
+
+mission "Merchant Start 3"
+	name "Special transport"
+	description "Get <bunks> travelers with <cargo> to <destination>"
+	passengers 2
+	cargo "traveling goods" 10
+	to offer
+		has "Merchant Start 2: done"
+	source
+		government "Republic" "Syndicate" "Free Worlds" "Neutral"
+	destination "New Switzerland"
+	npc
+		government "Pirate"
+		personality entering plunders
+		fleet "Bounty Hunters"
+	on offer
+		conversation
+			`"Captain," salutes the man, "This job involves <bunks> travelers trying to get to <destination> with <cargo> now, I have no idea why they need so much of it, but, get it over there alright?"`
+			choice	
+				`	"Yes, sir."`
+					accept
+				`	"Sorry, but I can't."`
+					decline
+	on complete
+		payment 50000
+		conversation
+			`You help the travelers lug his cargo away, still wondering why so much of it is needed. You see the first man walking up to you. "Well, Captain, may I ask you for your name?"`
+			choice
+				`	"It's <first> <last>."`
+					goto name
+				`	"I would rather not say."`
+					goto anonymous
+			label name
+			`	"Thank you, <first>"
+				goto next
+			label anonymous
+			`	The man smiles slightly. "That's perfectly fine, Captain <last>," he says, surprising you, "just do a better job of covering your pilot's license next time."`
+			label next
+			`	"Anyways, if you're still interested, meet me in the spaceport afterwards so we can converse further."`
+
+mission "Merchant Start 4"
+	name "Fast delivery"
+	description "Get <bunks> doctors to with <cargo> to <destination>
+	passengers 5
+	cargo "medical supplies" 15
+	deadline 13
+	to offer
+		has "Merchant Start 3: done"
+	source "New Switzerland"
+	destination "Moonshake"
+	npc
+		government "Pirate"
+		personality entering plunders
+		fleet "Bounty Hunters"
+		fleet "pirate raid"
+	on offer
+		conversation
+			`You find the man quietly sitting at a table drinking a cup of coffee.  He waves you over and signals to the barman to bring another drink. "Thank you for joining me, Captain <last>."
+			choice
+				`	"It's my pleasure."`
+					goto pleasantries
+				`	What's the mission?"
+					goto point
+			label pleasantries
+			`	He smiles, "Politeness can bring you places. There aren't many of your type in this galaxy."`
```

Comment:
Here and in the line directly below this code, the comma after "He smiles" should be a period instead, since "He smiles" is a independent clause.

`He smiles. "Politeness can bring you places..."`